src/components/components/general/button/IconButton.vue

```diff
@@ -0,0 +1,114 @@
+<template>
+	<button
+		class="c-application c-icon-button"
+		:class="[computedSize]"
+		v-bind="$attrs"
+		:disabled="disabled"
+		v-on="$listeners"
+		@click="$event.target.blur()"
+	>
+		<Icon :name="iconName" :color="computedColor" />
+	</button>
+</template>
+
+<script>
+import { buttonColors } from '@/components/components/general/button/Button';
+
+import Icon from '@/components/elements/core/icon/Icon';
+export const IconButtonSizes = ['small', 'medium', 'large'];
+
+export default {
```

Comment:
```suggestion
/**
 * @displayName c-icon-button
 */
export default {
```
컴포넌트에 c붙이는 작업을 하다가 다른 일 때문에 바빠서 못봐가지고 아직은 주석을 넣어줘야 돼요🥲

Reply:
@yooseongsil 이것도 놓쳤네요 ㅎㅎ